sim/src/main/cc/firesim/firesim_top.cc

```diff
@@ -299,6 +300,22 @@ firesim_top_t::firesim_top_t(int argc, char** argv)
     #endif
 #endif
 
+#ifdef DROMAJOBRIDGEMODULE_struct_guard
```

Comment:
There's only ever going to be one of these right?

Reply:
In the current setup, yes. Dromajo doesn't support multi-core simulation.